utils/utils/src/enum.ts

```diff
@@ -103,10 +104,21 @@ const statusToDisplayMappings: Record<TemplateStatus, string> = {
   VIRUS_SCAN_FAILED: 'Checks failed',
   WAITING_FOR_PROOF: 'Waiting for proof',
   PROOF_AVAILABLE: 'Proof available',
+  TEMPLATE_PROOF_APPROVED: 'Template proof approved',
 } as const;
 
-const templateStatusToDisplayMappingsLetter = (status: TemplateStatus) =>
-  statusToDisplayMappings[status];
+const isProofEmpty = (
```

Comment:
This returns `true` if `template.files.proofs` in non-empty, right?
Should this be called `isProofAvailable` instead?